internal/admin/exports/form.go

```diff
@@ -29,19 +29,20 @@ import (
 var ErrCannotSetBothTravelers = errors.New("cannot have both 'include travelers', and 'only non-travelers' set")
 
 type formData struct {
-	OutputRegion     string        `form:"OutputRegion"`
-	InputRegions     string        `form:"InputRegions"`
-	IncludeTravelers bool          `form:"IncludeTravelers"`
-	OnlyNonTravelers bool          `form:"OnlyNonTravelers"`
-	ExcludeRegions   string        `form:"ExcludeRegions"`
-	BucketName       string        `form:"BucketName"`
-	FilenameRoot     string        `form:"FilenameRoot"`
-	Period           time.Duration `form:"Period"`
-	FromDate         string        `form:"fromdate"`
-	FromTime         string        `form:"fromtime"`
-	ThruDate         string        `form:"thrudate"`
-	ThruTime         string        `form:"thrutime"`
-	SigInfoIDs       []int64       `form:"siginfo"`
+	OutputRegion       string        `form:"OutputRegion"`
+	InputRegions       string        `form:"InputRegions"`
+	IncludeTravelers   bool          `form:"IncludeTravelers"`
+	OnlyNonTravelers   bool          `form:"OnlyNonTravelers"`
+	ExcludeRegions     string        `form:"ExcludeRegions"`
+	BucketName         string        `form:"BucketName"`
+	FilenameRoot       string        `form:"FilenameRoot"`
+	Period             time.Duration `form:"Period"`
+	FromDate           string        `form:"fromdate"`
+	FromTime           string        `form:"fromtime"`
+	ThruDate           string        `form:"thrudate"`
+	ThruTime           string        `form:"thrutime"`
+	SigInfoIDs         []int64       `form:"siginfo"`
+	MaxRecordsOverride int           `form:"MaxRecordsOverride"`
 }
```

Comment:
it's too late for this, but weird how we've mixed lowercase and CamelCase. I tend to prefer case-insensitive if it's coming from web.

Reply:
We can change this pretty easily - it's an internal API